features/step_definitions/server.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
+
 const request = require('request');
 const Assert = require('chai').assert;
 
-module.exports = () => {
+module.exports = function server() {
```

Comment:
fat-arrow doesn't work so we had to change to this. It gets confused about the `this` in `this.Given` and `this.instance`